lwt/runtime.ml

```diff
@@ -46,11 +46,12 @@ let run_request
     (inp : M.input)
   =
   let meth, uri, headers =
-    Aws.Signing.sign_request ~access_key ~secret_key ~service:M.service ~region (M.to_http inp)
+    Aws.Signing.sign_request ~access_key ~secret_key ~service:M.service ~region (M.to_http M.service region inp)
   in
   let open Cohttp in
   let headers = Header.of_list headers in
   Lwt.catch (fun () ->
+    Printf.printf "HTTP %s: %s" (Aws.Request.string_of_meth meth) (Uri.to_string uri);
```

Comment:
@tmcgilchrist do you want the logging here?

Reply:
Really no, the `async` runtime does some logging and I added this mostly for debug reasons.
What it should do is accept a `verbose` option and log based off that. Larger problem and more effort, for another PR.

Reply:
Agreed